subworkflows/nf-core/fastq_qc_trim_filter_setstrandedness/main.nf

```diff
@@ -4,6 +4,7 @@ include { BBMAP_BBSPLIT                   } from '../../../modules/nf-core/bbmap
 include { CAT_FASTQ                       } from '../../../modules/nf-core/cat/fastq/main'
 include { SORTMERNA                       } from '../../../modules/nf-core/sortmerna/main'
 include { SORTMERNA as SORTMERNA_INDEX    } from '../../../modules/nf-core/sortmerna/main'
+include { FQ_LINT                         } from '../../../modules/nf-core/fq/lint/main'
```

Comment:
I see this change has not been made on nf-core/modules- it needs to be done there, and then the component added here